<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 22pt &quot;Uroob&quot;;</string>
   </property>
   <property name="text">
    <string>Калькулятор</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 16pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Первая цифра</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 16pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Вторая цифра</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
font: 17pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sub">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
font: 24pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mult">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
font: 17pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="div">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
font: 18pt &quot;Ubuntu&quot;;
font: 17pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 17pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Результат</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
